examples/oauth2/.env-sample

```diff
@@ -0,0 +1,20 @@
+# URI to fetch gateway configuration from. Expected format is: https?://[password@]hostname
```

Comment:
I didn't want to commit my .env since it would be specific to my environment, so I added a sample of what a .env file could look like. Could also be removed since this is documented in the README